### PR TITLE
Add full debug info for FxCop and fix CredScan issue

### PR DESCRIPTION
**Pull request by ChenYan98**

### CODE REVIEW

File: src/Microsoft.IdentityModel.Logging/Microsoft.IdentityModel.Logging.csproj [hunk 1]
```diff
@@ -12,6 +12,21 @@
     <PackageTags>.NET;Windows;Authentication;Identity;Logging</PackageTags>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
```
Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Logging/Microsoft.IdentityModel.Logging.csproj]: we can have a single group for debug, we don't need a separate group for the different targets.

File: build/VerifySigning.ps1 [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	$runtimes = $buildConfiguration.root.runtimes.Split(",")
+	
+	foreach ($project in $projects)
+	{
+		foreach ($runtime in $runtimes)
+		{
+			foreach ($pattern in $filePatterns)
+			{
+                $name = $project.name
+				$files = Get-ChildItem $srcPath\$name\bin\$buildType\$runtime\$pattern 2>&1
```
Comment (brentschmaltz) [inline build/VerifySigning.ps1]: fix the tabbing

File: build/VerifySigning.ps1 [hunk 3]
```diff
@@ -0,0 +1,92 @@
+param([string]$root, [string]$buildType="Debug")
+
+
```
Comment (brentschmaltz) [inline build/VerifySigning.ps1]: extra line, tabbing issues

File: src/Microsoft.IdentityModel.Logging/Microsoft.IdentityModel.Logging.csproj (hunk 1 above)
Comment (ChenYan98) [reply src/Microsoft.IdentityModel.Logging/Microsoft.IdentityModel.Logging.csproj]: fixed

File: build/VerifySigning.ps1 (hunk 2 above)
Comment (ChenYan98) [reply build/VerifySigning.ps1]: fixed

File: build/VerifySigning.ps1 (hunk 3 above)
Comment (ChenYan98) [reply build/VerifySigning.ps1]: fixed

File: src/Microsoft.IdentityModel.Tokens.Saml/Microsoft.IdentityModel.Tokens.Saml.csproj [hunk 4]
```diff
@@ -12,17 +12,7 @@
...
-    <DebugSymbols>true</DebugSymbols>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net451|AnyCPU'">
-    <DebugType>full</DebugType>
-    <DebugSymbols>true</DebugSymbols>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.4|AnyCPU'">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```
Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Tokens.Saml/Microsoft.IdentityModel.Tokens.Saml.csproj]: remove the |AnyCpu and then commit.

**Pull request merged**